### ISSUE TITLE
Accessibility a11y test fails

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#1143]
![image](https://user-images.githubusercontent.com/7963804/44907333-d4275280-ad17-11e8-9b5e-73488b68872f.png)

`> @ a11y /Users/marcoscastro/projects/clay
> metal-a11y --packages ./packages -c demos/a11y.html

events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at _errnoException (util.js:992:11)
    at TCP.onread (net.js:618:25)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ a11y: `metal-a11y --packages ./packages -c demos/a11y.html`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ a11y script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcoscastro/.npm/_logs/2018-08-31T10_17_17_662Z-debug.log`
